dev-server/static/scripts/util.js

```diff
@@ -1,13 +1,21 @@
+ /**
+  * Because the code in this file is not bundled by babel, it must be compatible
+  * with all the supported browsers version (see `browserlist` in `package.json`)
+  * without transpilation. Do not include latest EcmaScript features as these
+  * will cause exceptions while working on dev (`localhost:3000`) on slightly
+  * older, yet supported browser versions.
+  */
+
 /** @type {string|null} */
-export let activeClientUrl;
+var activeClientUrl;
```

Comment:
You can still use `let`/`const` rather than `var`.

Reply:
Yes, I agree.

I will change the all the `var` for `let` and merge the PR.